authors/apps/articles/renderers.py

```diff
@@ -0,0 +1,24 @@
+import json
+
+from rest_framework.renderers import JSONRenderer
+
+
+class ArticleJSONRenderer(JSONRenderer):
+    charset = 'utf-8'
+
+    def render(self, data, media_type=None, renderer_context=None):
```

Comment:
@frankopkusianwar, you can inherit from the BaseRenderer class created by @Rhytah  to remove the duplicate code